contracts/facets/GBMFacet.sol

```diff
@@ -128,9 +128,14 @@ contract GBMFacet is IGBM, IERC1155TokenReceiver, IERC721TokenReceiver, Modifier
     function claim(uint256 _auctionId) public override {
         address _ca = s.tokenMapping[_auctionId].contractAddress;
         uint256 _tid = s.tokenMapping[_auctionId].tokenId;
+        uint256 _tam = s.tokenMapping[_auctionId].tokenAmount;
 
         require(s.collections[_ca].biddingAllowed, "claim: Claiming is currently not allowed");
-        require(getAuctionEndTime(_auctionId) < block.timestamp, "claim: Auction has not yet ended");
+		if(s.auctionSeller[_auctionId] == address(0x0)){ //If it's a primary auction
```

Comment:
Primary auctions have their `s.auctionSeller` to be address(0)
However, uninitialized auctions(both primary & secondary) also have `s.auctionSeller` to be address(0)

Reply:
better to have another way to differentiate between priamary and secondary auctions